tests/next_tests/unit_tests/program_processor_tests/runners_tests/test_dace_fieldview.py

```diff
@@ -776,6 +785,447 @@ def test_gtir_connectivity_shift_chain():
     assert np.allclose(e_out, ref)
 
 
+def test_gtir_neighbors_as_input():
+    # FIXME[#1582](edopao): Enable testcase when type inference is working
```

Comment:
@tehrengruber I have added 6 testcases for `neighbors` and `reduce` but 3 of them are currently skipped, because of the issues we discussed about fields of lists as input/output to programs or in combination with map operations. Later we can try to enable them, when GTIR type inference has support.